semilla_moneda/semilla_moneda.py

```diff
@@ -28,8 +28,7 @@ def main():
     )
 
     ### CONVERSIONES Y HASH
-    random_hex = f"{int(binario, 2):x}"
-    random_bin = bytes.fromhex(random_hex)
+    random_bin = int(binario, 2).to_bytes(binario_len // 8, byteorder = 'big')
```

Comment:
I think your approach is better than the current one, but given you're defining the length based on the length of the binary and performing an integer division, you'd need to make sure that the input is of the expected lengths, otherwise, this could lead to off-by-one errors.

Reply:
Sure, I've added a new commit https://github.com/lunaticoin/semilla-moneda/pull/2/commits/053f2989094c9763e9c5a6c04d36d4ae18b8442f that handles this, can you take look at it and let me know?